controllers/datadogagent/utils.go

```diff
@@ -650,6 +650,13 @@ func getEnvVarsForAgent(dda *datadoghqv1alpha1.DatadogAgent) ([]corev1.EnvVar, e
 					Value: datadoghqv1alpha1.ClusterAndEndpointsConfigPoviders,
 				})
 			} else {
+				// Remove ksm v1 conf if the cluster checks are enabled and the ksm core is enabled
+				if isKSMCoreEnabled(spec.ClusterAgent) {
+					envVars = append(envVars, corev1.EnvVar{
```

Comment:
If the user provides their own ignore list - are we going to replace it with just one containing `kubernetes_state`?

Reply:
hum, my comment did not make it, it seems... I thought appending to the envvar list was enough but you are right, I need to append to the value of the var